instagram-app/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import PostContainer from "./components/PostContainer"
+import SearchBar from "./components/SearchBar"
+
+
+class App extends React.Component {
+  constructor(){
+    super()
+      this.state={
+        data:dummyData,
+       }
+    }
+    
+
+  render() { 
+    return ( 
+      <div className="App">
+      <SearchBar />
```

Comment:
You might want to pass an input prop into the <SearchBar/> component - you will need a way to store the search input text in your app so that you can use it in your application state later on for making your search bar functional.